brayns/parameters/GeometryParameters.cpp

```diff
@@ -194,7 +199,12 @@ GeometryParameters::GeometryParameters()
         //
         (PARAM_CIRCUIT_MESH_TRANSFORMATION.c_str(),
          po::bool_switch()->default_value(false),
-         "Enable mesh transformation according to circuit information.");
+         "Enable mesh transformation according to circuit information.")
+        //
+        (PARAM_DEFAULT_BVH_FLAG.c_str(),
+         po::value<std::vector<std::string>>()->multitoken(),
+         "Sets one or more property flags to be used by default when creating "
+         "a bvh [dynamic|compact|robust]");
```

Comment:
BVH
Same as before, it's not clear that this is a list? Or does boost program_options take care of indicating it?

Reply:
I don't know how to explain it without a wall of text.

Reply:
"Set a default flag to apply to BVH creation, one of  [dynamic|compact|robust], may appear multiple times."

I'm not sure whether --default-bvh-flag "dynamic robust" is also accepted though.

Reply:
It is.

Reply:
Then, the message is not totally accurate, but at least it doesn't miss the possibility of setting multiple flags.